testing/browserpass/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=browserpass
+pkgver=3.0.6
+pkgrel=0
+pkgdesc="Browser extension for pass"
+url="https://github.com/browserpass/browserpass-native"
+arch="all"
+license="MIT"
+depends="pass"
+makedepends="go git"
+options="!check" # no tests to be run
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/browserpass/browserpass-native/archive/$pkgver.tar.gz"
+builddir="$srcdir/github.com/browserpass/browserpass-native"
+
+prepare() {
+	mkdir -p "${builddir%/*}"
+	mv "$srcdir/browserpass-native-$pkgver" "$builddir"
+	default_prepare
+	sed -i "s:%%replace%%:/usr/bin/browserpass-native:" \
+			browser-files/chromium-host.json browser-files/firefox-host.json
+}
+
+build() {
+	export GOPATH="$startdir"
+	go get -v -d
```

Comment:
I don't see any vendored libs or other kind of lock file. This means that dependencies can change any moment and the build is not reproducible, which is required for alpine packages.

Reply:
Hi, I'm using "Go modules", the lock file is called `go.sum`, and Go team [claims them to be reproducible](https://github.com/golang/go/wiki/Modules#modules):

> Modules record precise dependency requirements and create reproducible builds.

Reply:
Thanks for the info. I'm trying to keep up with all the go package managers :-)

Reply:
Heh, same here :P

Reply:
Hopefully this is the last package manager to learn, since it's built into Go itself 😄